packages/peregrine/lib/store/actions/cart/__tests__/asyncActions.spec.js

```diff
@@ -131,8 +131,8 @@ describe('createCart', () => {
             user: { isSignedIn: false }
         }));
 
-        const error = 'AN ERROR FROM GQL';
-        fetchCartId.mockResolvedValue({ error });
+        const errors = ['AN ERROR FROM GQL'];
+        fetchCartId.mockResolvedValue({ errors });
```

Comment:
We should add another test to cover the throw use case; almost exactly this but with `mockRejectedValueOnce(new Error('Woof'))`.